### PR TITLE
Added PII Parser Module

### PR DESCRIPTION
**Pull request by utkusen**
This pull request includes my `PIIParser` module which helps to identify PII data located inside Word and Excel files. The PII regexes can be changed inside `Get-PII.ps1` file. Currently it detects: All types of credit card numbers, SSN, Passport number, Birthday, Ip Address, Turkish Identification Number

To use the module inside Empire, navigate to powershell/modules/collection/PIIParser. You need to set `FilePath` value which represents fullpath of target document. For example: `C:\Users\utku\document.docx`. It returns first few characters of detected data, rest is remain masked.

### CODE REVIEW

**Comment by vinnybod** [comment]
Thank you for your pull request!

We're working on bringing in [Python3 compatability](https://github.com/BC-SECURITY/Empire/pull/3) at the moment and don't want to introduce any new complexities during that transition. I expect that we'll be ready to merge in a change like this in the next couple weeks.

I'll be updating the readme with more information in the future, but all pull requests coming in will need to be compliant with [both Python 2.7 and Python 3](https://python-future.org/compatible_idioms.html)

File: lib/modules/powershell/collection/PiiParser.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
+from lib.common import helpers
```
Comment (roseaj) [inline lib/modules/powershell/collection/PiiParser.py]: Add `from __future__ import print_function` and `from builtins import str`

File: lib/modules/powershell/collection/PiiParser.py [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+                    self.options[option]['Value'] = value
+
+
+    def generate(self, obfuscate=False, obfuscationCommand=""):
+
+        moduleSource = self.mainMenu.installPath + "/data/module_source/collection/Get-PII.ps1"
+        try:
+            f = open(moduleSource, 'r')
+        except:
+            print helpers.color("[!] Could not read module source path at: " + str(moduleSource))
```
Comment (roseaj) [inline lib/modules/powershell/collection/PiiParser.py]: Update to `print (helpers.color("[!] Could not read module source path at: " + str(moduleSource)))`

File: lib/modules/powershell/collection/PiiParser.py [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+            return ""
+
+        script = f.read()
+
+        scriptEnd = '$global:FilePath = '
+
+        for option,values in self.options.iteritems():
```
Comment (roseaj) [inline lib/modules/powershell/collection/PiiParser.py]: `iteritems` should be replaced with `items`

**Comment by Cx01N** [comment]
Closing due to inactivity. Please reopen if you are able to resolve the conflicts.

**Pull request closed without merging**